game/static/game/css/theme.css

```
@@ -195,3 +195,321 @@
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+    padding: 1.5rem 5%;
+    background-color: var(--nav-bg);
+    border-bottom: 1px solid var(--border-color);
+    position: sticky;
+    top: 0;
+    z-index: 1000;
+    margin: 14px 22px;
+    border-radius: 18px;
+    background: rgba(15, 15, 25, 0.55);
```

Stylelint (17.14.0)

[error] Overridden property "background-color" by shorthand "background" (declaration-block-no-shorthand-property-overrides)

(declaration-block-no-shorthand-property-overrides)